src/backend/cuda/kernel/sift_nonfree.hpp

```diff
@@ -1108,8 +1109,7 @@ std::vector<Array<T>> buildGaussPyr(Param<T> init_img, const unsigned n_octaves,
                 tmp_pyr.push_back(
                     createEmptyArray<T>({tmp_pyr[src_idx].dims()[0] / 2,
                                          tmp_pyr[src_idx].dims()[1] / 2}));
-                resize<T, InterpolationType::Bilinear>(tmp_pyr[idx],
-                                                       tmp_pyr[src_idx]);
+                resize<T>(tmp_pyr[idx], tmp_pyr[src_idx], AF_INTERP_BILINEAR);
```

Comment:
thanks for this  fix. I must have forgotten to turn on build-non-free when I switched computers while fixing these in my PR.